### DISCUSSION TITLE
Get.to() with custom transition, possible?

### DISCUSSION THREAD

**Discussion by rooque** [jonataslaw/getx#2190] [Q&A] [answered]
Hello 👋

Does anyone know if using Get.to() with custom transition is possible?  🤔

If it's possible, please provide an example  📄✍

**Comment by ashunasar** [comment]
you can use transition by just doing this

```
         Get.to(HomeScreen(),transition: Transition.rightToLeftWithFade);

```

**Reply by offstd** [reply]
Sure, but is it possible to write my own Transition animation? Extend the transitions provided by GetX?

**Comment by sanjidbillah** [comment, marked as answer]
You can't make custom transition out of this list.
 
https://github.com/jonataslaw/getx/blob/6ab6adb731c503f75827c90677f90b0dd35d93da/lib/get_navigation/src/routes/transitions_type.dart#L5-L22

**Reply by rooque** [reply]
That's what I thought, maybe in the next versions ...
